R/derive_param_exist_flag.R

```diff
@@ -179,6 +187,7 @@ derive_param_exist_flag <- function(dataset = NULL,
                                     filter_add = NULL,
                                     aval_fun,
                                     by_vars = get_admiral_option("subject_keys"),
+                                    keep_source_vars = exprs(everything()),
```

Comment:
Here we have the same problem as for `derive_var_merged_exist_flag()`: the result depends on the sort order. We could add the `order` and `mode` argument to the function. But this would make the function very similar to `derive_extreme_records()`. So I am not sure if this is a good idea.

For now I would not add `keep_source_vars` to `derive_param_exist_flag()`.

I think we need to discuss in the clean-up group if we need the `derive_*_exist_flag()` functions and in the affirmative how they can be clearly distinguished from other functions like `derive_extreme_records()`.